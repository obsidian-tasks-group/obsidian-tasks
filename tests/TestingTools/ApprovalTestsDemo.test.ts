import { verify, verifyAsJson } from 'approvals/lib/Providers/Jest/JestApprovals';

describe('ApprovalTests', () => {
    // begin-snippet: approval-test-as-text
    test('SimpleVerify', () => {
        verify('Hello From Approvals');
    });
    // end-snippet

    // begin-snippet: approval-test-as-json
    test('JsonVerify', () => {
        const data = { name: 'fred', age: 30 };
        verifyAsJson(data);
    });
    // end-snippet
});
