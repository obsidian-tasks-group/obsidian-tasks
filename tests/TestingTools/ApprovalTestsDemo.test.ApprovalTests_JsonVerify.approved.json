{
  "name": "fred",
  "age": 30
}
